lib/src/components/NxNavigationSidebar/NxNavigationSidebar.scss

```diff
@@ -0,0 +1,146 @@
+/**
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+@import '../../scss-shared/nx-variables';
+@import '../../scss-shared/nx-text-helpers';
+@import '../../scss-shared/nx-container-helpers';
+
+$nx-sidebar-max-width: 300px;
+$nx-sidebar-width-open: 250px;
+$nx-sidebar-width-closed: 72px;
+$nx-sidebar-logo-height: 34px;
+
+$nx-sidebar-text-color: $nx-white;
+$nx-sidebar-background-color: $nx-blue-1000;
+
+$nx-sidebar-active-color: $nx-indigo-400;
+$nx-sidebar-selected-color: $nx-indigo-800;
+$nx-sidebar-hover-color: $nx-indigo-600;
+
+$nx-sidebar-focus-border: 1px solid $nx-focus-border-color;
+
+$nx-sidebar-separator-color: $nx-grey-900;
+
+.nx-page-sidebar-operable {
+  @include container-vertical;
+
+  background-color: $nx-sidebar-background-color;
+  box-sizing: border-box;
+  color: $nx-sidebar-text-color;
+  max-width: $nx-sidebar-max-width;
+  padding: $nx-spacing-xl $nx-spacing-l $nx-spacing-l $nx-spacing-l;
+  width: $nx-sidebar-width-open;
+
+  .nx-text-link {
+    color: $nx-sidebar-text-color;
+  }
+}
+
+.nx-page-sidebar-operable__header {
+  display: flex;
+  margin-bottom: $nx-spacing-xl;
+}
+
+.nx-page-sidebar-operable__toggle {
+  border-color: transparent;
+  border-radius: 50%;
+  color: $nx-sidebar-text-color;
+  flex: none;
+  height: 34px;
+  margin: 0;
+  width: 34px;
+
+  &:focus {
+    border-color: $nx-focus-border-color;
+  }
+
+  &:active {
+    background-color: $nx-sidebar-active-color;
+  }
+
+  &:hover {
+    background-color: $nx-sidebar-hover-color;
+  }
+}
+
+.nx-page-sidebar-operable__product-info {
+  flex: 1 1 auto;
+  height: $nx-sidebar-logo-height;
+}
+
+.nx-sidebar-navigation {
+  border-bottom: 1px solid $nx-sidebar-separator-color;
+  flex: none;
+
+  margin: 0 -#{$nx-spacing-l} $nx-spacing-md -#{$nx-spacing-l};
+  padding-bottom: $nx-spacing-xxl;
+}
+
+.nx-sidebar-navigation__link {
+  @include container-horizontal;
+
+  align-items: center;
```

Comment:
I'd feel more comfortable if the icon/text alignment was being done using inline layout not flex with `align-items: center`.  What you've got now seems like it's only lined up correctly by coincidence.  I've proven out what I mean here: https://github.com/sonatype/sonatype-react-shared-components/commit/28b900bd4ebae9c96d45b52f994d09886e6bbe54

Reply:
> I'd feel more comfortable if the icon/text alignment was being done using inline layout not flex with `align-items: center`. What you've got now seems like it's only lined up correctly by coincidence. I've proven out what I mean here: [28b900b](https://github.com/sonatype/sonatype-react-shared-components/commit/28b900bd4ebae9c96d45b52f994d09886e6bbe54)

Done in 3622274d9846ff062395c8d3ffc442b81b40f42a